### PR TITLE
feat: Run cypress on documentation site periodically

### PR DESCRIPTION
**Pull request by ntt261298**
## Related issues

- #181 

## Features

- [x] Run cypress on documentation site periodically (2 times per day) using github action

### CODE REVIEW

File: .github/workflows/cypress.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name: End-to-end tests
+on:
+  schedule:
+    # Runs at 10h and 22h everyday
+    - cron: '0 10,22 * * *'
```
Comment (nvh95) [inline .github/workflows/cypress.yml:5]: I guess this one needs to be merged to take effect 🤔

Comment (nvh95) [reply .github/workflows/cypress.yml:5]: If so, please just merge it on your own.

Comment (ntt261298) [reply .github/workflows/cypress.yml:5]: I merged it into main

Comment (ntt261298) [reply .github/workflows/cypress.yml:5]: Seems like it doesn't work now, will take a look.

Comment (nvh95) [reply .github/workflows/cypress.yml:5]: @ntt261298 You can try to test in "on push" before running periodically.

**Pull request merged**